libs/wordpress-plugin/plugin/trunk/server/query.php

```diff
@@ -51,9 +51,9 @@ function block_protocol_encode_filter_value($value)
   if (is_int($value)) {
     return "{$value}";
   } elseif (is_string($value)) {
-    return "'{$value}'";
+    return "\"{$value}\"";
```

Comment:
This made a big difference for some reason :shrug: 
Without it MariaDB would fail on strings.